newton/_src/viewer/viewer_viser.py

```
@@ -1295,23 +2266,62 @@ def log_points(
+                # when active-particle compaction changes the cloud length.
+                base_color = cached_colors[0] if len(cached_colors) else np.full(3, 255, dtype=np.uint8)
+                colors_val = np.asarray(base_color, dtype=np.uint8)
+            try:
+                existing_handle.points = points_val
+                self._set_handle_property_if_changed(existing_handle, "point_size", point_size)
+                if colors_val is not None and not np.array_equal(cached_colors, colors_val):
+                    existing_handle.colors = colors_val
+                    self._point_cloud_colors[name] = colors_val.copy()
+                self._set_handle_property_if_changed(existing_handle, "visible", True)
+                return
+            except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)